ext/msgpack/unpacker.c

```diff
@@ -60,13 +60,16 @@ void msgpack_unpacker_static_destroy()
 #define HEAD_BYTE_REQUIRED 0xc1
 
 static inline msgpack_unpacker_stack_t* _msgpack_unpacker_new_stack(void) {
+    msgpack_unpacker_stack_t *stack = ZALLOC(msgpack_unpacker_stack_t);
```

Comment:
We have to do one more alloc, which is a bit unfortunate. In theory we could alloc the stack header and stack data together, but I'd need to dig into how `msgpack_rmem_alloc` works.